lms/static/hawcons.css

```diff
@@ -1181,6 +1181,9 @@
 .hawcons-icon-one-finger-swipe-down:before {
 	content: "\e782";
 }
+span[class*="hawcons-icon-file"] {
```

Comment:
This may cause long rendering time.

Instead, add class to the template file. The selector should be `span.file-icon`.